event_tracker/event_tracker_resource.tres

```diff
@@ -0,0 +1,7 @@
+[gd_resource type="Resource" script_class="Event_Tracker" load_steps=2 format=3 uid="uid://bmwmiov8jhr7n"]
```

Comment:
I don't think we need a resource for this, just using the signal pattern I described should be fine